src/test/resources/features/06_pensions.feature

```diff
@@ -1114,7 +1114,7 @@ Feature: Pensions Journeys - Income Tax Submission
     When the user clicks the save and continue button
     Then the user should be redirected to the "Pensions" page
 
-
+  @ignore
```

Comment:
Could you maybe add a comment to see comments here https://jira.tools.tax.service.gov.uk/browse/SASS-7078